stream_alert_output/main.py

```diff
@@ -120,7 +121,10 @@ def run(self, alerts):
                 output_func = getattr(self, '_{}'.format(output), None)
                 if output_func:
                     self._setup_output_creds(output)
-                    output_func(rule_name, formatted_alerts)
+                    try:
```

Comment:
let's keep it how it was initially, otherwise the error raised would never actually fire